README.rst

```diff
@@ -191,13 +238,21 @@ This auth token can be supplied in the Authorization header.
 Message Broker
 ==============
 
-RabbitMQ is used to broker messages between **cloudigrade** and inspectigrade services. There are multiple Python packages available to interact with RabbitMQ; the officially recommended packaged is `Pika <https://pika.readthedocs.io/en/latest/>`_. Both services serve as producers and consumers of the message queue. The **cloudigrade** docker-compose file requires that a password environment variable be set for the RabbitMQ user. Make sure that the following has been set in your local environment before starting
+RabbitMQ is used to broker messages between **cloudigrade** and inspectigrade
+services. There are multiple Python packages available to interact with
+RabbitMQ; the officially recommended packaged is `Pika
+<https://pika.readthedocs.io/en/latest/>`_. Both services serve as producers
+and consumers of the message queue. The **cloudigrade** docker-compose file
+requires that a password environment variable be set for the RabbitMQ user.
+Make sure that the following has been set in your local environment before
+starting
```

Comment:
I did not see any information before this section about installing or setting up RabbitMQ. 
Does there need to be a section about that? Does it allready exist somewhere else?

Reply:
Since we only very recently started actively using RabbitMQ, we may need to expand this. I'm taking a note to do that.

FWIW, as long as you're using docker-compose, you don't need to install anything locally. We just pull a public container image and use that.